agent/provider/src/market/termination_reason.rs

```diff
@@ -29,11 +32,37 @@ pub enum BreakReason {
     )]
     #[strum(message = "DebitNotesDeadline")]
     DebitNotesDeadline(chrono::Duration),
+    #[display(fmt = "Requestor rejected the debit note: {:?}", _0)]
```

Comment:
How will this be printed? This is debug formatting so I don't expect anything pleasant..

Reply:
https://github.com/golemfactory/ya-client/blob/4c304836de47be0633f13246da96023979a313f8/model/src/payment/rejection.rs

Reply:
Please display only information that are really useful in this context and avoid debug formatting. This message will be passed to Agreement termination reason, so should be pretty

Reply:
I've changed this line to log only the `rejection_reason` from `Rejection`. `RejectionReason` is a plain enum (so will only result in variant names) but doesn't implement `Display`. 

This enum originates from `ya_client` - I'd rather not implement a custom `to_string` but modify the target crate instead. As of now, the message shout be looking ok